examples/st7789-multiple-lcds.rs

```diff
@@ -0,0 +1,88 @@
+//!
+//! Demonstrates use of the Flexible Static Memory Controller to interface with four ST7789 LCD
+//! controllers
+//!
+//! Hardware required: Some board with 4 LCDs all connected to the microcontroller's flexible static
+//! memory controller
+//!
+//! We don't know if hardware like that actually exists. Still, this example shows how to compile
+//! code that uses multiple address pins and multiple chip select pins.
+//!
```

Comment:
I'm also no big fan of theoretical examples, I think every example should provide something of immediate value to a potential user.